toast.nim

```diff
@@ -146,6 +151,7 @@ when isMainModule:
     "defines": "definitions to pass to preprocessor",
     "includeDirs": "include directory to pass to preprocessor",
     "symOverride": "skip generating specified symbols",
+    "pluginFile": "Nim file to build and load as a plugin",
```

Comment:
similar to a previous comment, renaming to `"pluginSourcePath"` makes it clear it refers to source (nim) instead of compiled (dylib)